src/appRedux/publisherConfigSlice.ts

```diff
@@ -13,7 +17,12 @@ export const initialState: PublisherConfigInitialState = {
   termsUrl: '',
   resetUrl: '',
   enable3DSRedirectFlow: false,
-  isPaymentCheckboxDisabled: false
+  isPaymentCheckboxDisabled: false,
+  googleRecaptcha: {
+    showCaptchaOnPurchase: false,
+    showCaptchaOnRegister: false,
+    sitekey: import.meta.env.VITE_RECAPTCHA_SITE_KEY
```

Comment:
@pawelacio , please add the respective site key to the following files with the right value for each environment (probably our site key or an empty string if we don't want to enable captcha, e.g. for staging)
`.env.production`
`.env.sandbox`
`.env.staging`

currently we don't have these values set, so the evaluation here would resolve to `undefined`

thanks!

Reply:
ah ok, closing this - i've just read all the other messages from GitHub and Chat.